routing/index_graph_starter.cpp

```diff
@@ -62,6 +74,84 @@ m2::PointD const & IndexGraphStarter::GetPoint(RoadPoint const & rp)
   return GetPoint(FindFakeEdge(rp.GetFeatureId()).GetFrom());
 }
 
+void IndexGraphStarter::GetOutgoingEdgesList(TVertexType const & u, vector<TEdgeType> & edges)
+{
+  vector<JointEdge> jes;
```

Comment:
To be consistent, if nothing else. Either notation is fine as long as they are not mixed for no reason.